kodilla-stream/src/main/java/com/kodilla/stream/forumuser/Forum.java

```diff
@@ -0,0 +1,27 @@
+package com.kodilla.stream.forumuser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class Forum {
+
+    private final List<ForumUser> theUsersList = new ArrayList<>();
+
+    public Forum() {
+        theUsersList.add(new ForumUser(1, "random1", 'M', 1990, 3, 12, 10));
```

Comment:
takie rzeczy w konstruktorze klasy? :o
Konstruktor powinien orzymać listę, zaś taka inicjalizacja powinna sie odbyć w klasie StreamMain

Reply:
Właśnie tak było w przykładach kursowych i też mnie to zdziwiło. Dlatego u mnie tak wstawiłem, bo chciałem zobaczyć czy zwrócisz uwagę, że się tak nie robi ;)